From e803c93bc68ce4e2037465072bbd5fab559f90d9 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Sat, 21 Feb 2026 21:54:12 +0100
Subject: [PATCH] ENH: Add Markups MRML include directories to Logic

Add vtkSlicerMarkupsModuleMRML include directories to Sequences Logic
module build configuration.
---
 Modules/Loadable/Sequences/Logic/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/Loadable/Sequences/Logic/CMakeLists.txt b/Modules/Loadable/Sequences/Logic/CMakeLists.txt
index 5f4c591549..58fc813cc1 100644
--- a/Modules/Loadable/Sequences/Logic/CMakeLists.txt
+++ b/Modules/Loadable/Sequences/Logic/CMakeLists.txt
@@ -5,6 +5,7 @@ set(KIT ${PROJECT_NAME})
 set(${KIT}_EXPORT_DIRECTIVE "VTK_SLICER_${MODULE_NAME_UPPER}_MODULE_LOGIC_EXPORT")
 
 set(${KIT}_INCLUDE_DIRECTORIES
+  ${vtkSlicerMarkupsModuleMRML_INCLUDE_DIRS}
   )
 
 set(${KIT}_SRCS
-- 
2.52.0

